tutorials/vue/axios-put.md

```diff
@@ -0,0 +1,48 @@
+With a few lines of code, you can easily make an
+[`axios.put()`](/tutorials/axios/put) request with Vue:
```

Comment:
I'd make a note that we recommend putting `axios.put()` calls in methods or [mounted hooks](https://masteringjs.io/tutorials/vue/mounted).